src/test/java/ReceiptAnalysisTest.java

```diff
@@ -221,6 +233,48 @@ public void analyzeImageAtUrlReturnsDateIn1900s() throws IOException, ReceiptAna
     Assert.assertEquals(TIMESTAMP_IN_1900S, results.getTimestamp());
   }
 
+  @Test
+  public void analyzeImageAtUrlReturnsPrice() throws IOException, ReceiptAnalysisException {
```

Comment:
Can you rename the tests so they have the format methodBeingTested_situationBeingTested_expectedResult().

Example: analyzeImageAtUrl_rawTextWithPrice_returnsPrice()

Reply:
I talked with Tony about this and he said that the current names seem to include the required information. Thus, I don't want to spend time switching the format of every test method name unless it's really necessary.

Reply:
I've updated the new tests to use this format. The rest should be handled in #124.